lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -41,6 +41,67 @@
             }
         });
 
+        LearnerProfileFieldViews.ProfileImageFieldView = FieldViews.ImageFieldView.extend({
+
+            titleAdd: gettext("upload a photo"),
+            titleEdit: gettext("change photo"),
+
+            imageUrl: function () {
+                return this.model.profileImageUrl();
+            },
+
+            imageAltText: function () {
+                return interpolate_text(gettext("Profile photo for {username}"), {username: this.model.get('username')});
```

Comment:
Sorry, I think I missed this before. I'd change this to "Profile image for {username}"